src/Net/Http.fs

```diff
@@ -681,16 +1404,14 @@ module private HttpHelpers =
             mimeType.Split([| ';' |], StringSplitOptions.RemoveEmptyEntries)
             |> Array.exists isText
 
-        use stream = stream
-        let! memoryStream = asyncRead stream
-
-        let memoryStream = 
+        use! memoryStream = asyncRead stream
+        use memoryStream =
```

Comment:
Shouldn't we keep this one as a let? There is already a use inside decompress, and on the else without unzipping then you would be calling dispose twice. I'll fix it while merging